src/pop_protocol.erl

```diff
@@ -167,19 +168,6 @@ add_new_block(Block, CurrentTime, ProtocolData)
 
     ?assert(PreviousBlockTimestamp < Tmp, "time should be larger than previous"),
     ?assert(Tmp - TimeDesyncMargin < CurrentTime, "block cannot be in the future"),
```

Comment:
btw Ivy told me ?asserting in erlang code isn't idiomatic.  (I sent her the potato paper and she snooped around our code)

of course she also didn't tell me what WAS idiomatic 😱

Reply:
💯